### ISSUE TITLE
how to return  class label from the module

### ISSUE THREAD

**Issue by DuyguA** [tensorflow/tfjs#2705]
#### TensorFlow.js version

tensorflow/tfjs-node: 1.2.11

#### Browser version

No browser, Typescript project.

#### Describe the problem or feature request

I trained an LSTM with Keras Python. I can successfully load the model into Node.js , output `predict`  result (class label) to the console but I cannot return this label to the upstream modules. Instead as a return value, I get a `Promise<any>` .  I cannot break the chain of `Promise`s and get the prediction value.

#### Code to reproduce the bug / link to feature request

```
async function predict(sentence: string) {
  const model = await tfnode.loadLayersModel(
    "file://./Model/minimal_model_dir/model.json"
  );
  console.log(model.summary());  //outputs the model summary
  const dummy = tf.ones([1, 15]);
  const classLabel = model.predict(dummy);
  console.log(classLabel.dataSync());  //outputs the value
  return classLabel.dataSync();
}


upstream module:

const prediction = predict(sentence)  // Here I cannot get the value, prediction type is Promise<any>
```

I tried

```
let result;
predict(sentence).then(val => {result = val});
```
but when I output the `result` it was `undefined` . How can I assign the prediction result into a local or gloabal variable?

**Comment by rthadur** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow.js) since it is not a bug or feature request. There is also a larger community that reads questions there.

**Comment by DuyguA** [comment]
@rthadur  , can you explain to the audience why there is a label called `type:support` then? 
I don't think your thinking is optimal, please reopen the issue.
